examples/technical_blogs/tartan_blog/build_and_run_networks.py

```diff
@@ -0,0 +1,156 @@
+import subprocess
+import os
+import shutil
+import requests
+
+# Prereqs: portBLAS, SYCL-DNN, oneapi oneapi_construction_kit should be cloned and installed.
+portBLAS_source = os.environ.get("portBLAS_source")
```

Comment:
Having these as arguments would be more user-friendly. I don't know how much of a priority that is for this script.

Using `argparse`, for example, would allow users to call `--help` and see what arguments they're meant to be supplying. Especially if these environment variables are only used as script inputs, which they appear to be.